test/test-script.js

```diff
@@ -387,44 +387,48 @@ describe('Player', function() {
   }); // end .equippedWith specs
 }); // end Player specs
 
+var zombie
+
 describe('Zombie', function() {
+  beforeEach(function() {
+    zombie = new Zombie(30, 10, 5);
+  });
   it('should be a function', function() {
     expect(Zombie).to.be.a('function');
   });
 
   it('should have health', function() {
-    var zombie = new Zombie(30, 10, 5);
     zombie.health.should.equal(30);
   });
 
   it('should have a strength attribute', function() {
-    var zombie = new Zombie(30, 10, 5);
     zombie.strength.should.equal(10);
   });
 
   it('should have a speed attribute', function() {
-    var zombie = new Zombie(30, 10, 5);
     zombie.speed.should.equal(5);
   });
 
   it('should have a private maxHealth variable', function() {
-    var zombie = new Zombie(30, 10, 5);
     expect(zombie.maxHealth).to.be.undefined;
   });
 
   it('should be alive', function() {
-    var zombie = new Zombie(30, 10, 5);
     zombie.isAlive.should.equal(true);
   });
 }); // end Zombie specs
 
+var charger;
```

Comment:
`var charger`; should go inside `describe('FastZombie'...`
just before `beforeEach`